content/awerv/drone-pushover-plugin/index.md

```diff
@@ -0,0 +1,54 @@
+---
+version: '0.2'
+date: 2020-07-17T00:00:00+00:00
+title: Pushover
+author: awerv
+tags: [ Pushover, notifications ]
+logo: pushover.svg
+repo:  awerv/drone_pushover_plugin 
```

Comment:
Does it really use underscores in the name?

Reply:
Yes, it does. Is it a big problem?